src/app/clusters/ota-requestor/DefaultOTARequestorDriver.h

```diff
@@ -65,11 +65,11 @@ class DefaultOTARequestorDriver : public OTARequestorDriver
     bool CanConsent() override;
     uint16_t GetMaxDownloadBlockSize() override;
     void SetMaxDownloadBlockSize(uint16_t maxDownloadBlockSize) override;
-    void HandleError(UpdateFailureState state, CHIP_ERROR error) override;
```

Comment:
Since we're getting rid of this API, is there a need to keep `UpdateFailureState` enum class around at all. I know DefaultOTARequestor calls it whenever it calls `RecordErrorUpdateState` but I believe all that function does with the value is pass it into `HandleError`.

Reply:
As we discussed, even though `UpdateFailureState` may be useful in the future, we will remove it for now as it is dead code and put it back in if we need the `UpdateFailureState` info in the future.